go.mod

```diff
@@ -7,7 +7,7 @@ module github.com/pingcap/tidb-operator
 go 1.21
 
 require (
-	cloud.google.com/go/storage v1.10.0
+	cloud.google.com/go/storage v1.30.1
```

Comment:
Why there is go.mod change?

Reply:
upgrade to `kvproto v0.0.0-20231122054644-fb0f5c2a0a10`,
which imports `cloud.google.com/go v0.110.10`,
which imports `cloud.google.com/go/storage v1.30.1`

`go.mod` use the newest version of the package.